website/cue/reference/components/sinks/vector.cue

```diff
@@ -80,7 +85,7 @@ components: sinks: vector: {
 			required:    true
 			warnings: []
 			type: string: {
-				examples: ["92.12.333.224:5000"]
+				examples: ["92.12.333.224:\(_port)"]
```

Comment:
Aren't we missing the `version` configuration documentation here?

When we add it, I'd add a `warning` about the v1 / v2 incompatibilities and possible message loss for v1 sink -> v2 source.

Reply:
Good catch! Added in 0d8523ea36d6b1a03ad73f3d845b466965a9f231